src/oc/validation-inline.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines contain the default error messages used by
+    | the validator class. Some of these rules have multiple versions such
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+
+    'accepted'             => 'Lo camp :attribute deu èsser acceptat.',
```

Comment:
The file with the "inline" suffix does not use the passing of the attribute name. This must be done in a "regular" file.

Reply:
I really don't understand the difference between the two files...
validation-inline is shorter?

Reply:
Hum, why are there more string in the FR folder? like this:
  'attributes' => [
        'name'                  => 'nom',
        'username'              => "nom d'utilisateur",
        'email'                 => 'adresse email',
        'first_name'            => 'prénom',
        'last_name'             => 'nom',
        'password'              => 'mot de passe',
        'password_confirmation' => 'confirmation du mot de passe',
        'city'                  => 'ville',
        'country'               => 'pays',

Reply:
> I really don't understand the difference between the two files...
> validation-inline is shorter?

Inline is used for SPA, when you need to display a list of errors not as a list, but for a specific field. Therefore, in them we write "this field", and not "The Foo field".

> Hum, why are there more string in the FR folder? like this:
@caouecs, need help.

Reply:
Hello,
occitania, very good idea, I'm from Toulouse ;)

In French in the text:

* validation.php : c'est le fichier officiel de Laravel avec les tags :attribute dans les traductions
* validation-inline.php : c'est un fichier que la communauté a demandé pour avoir des textes plus génériques sur la validation, sans :attribute. Les deux fichiers se ressemblent effectivement, mais nous avons besoin des deux.
* dans validation.php, il y a une partie *attributes*, c'est une partie facultative, aucune obligation de traduire.

J'espère que vous êtes un occitan de France, et que vous avez compris mes explications. Sinon, je peux tenter en espagnol ; désolé mais mon occitan est assez faible, ma famille est plus catalan et basque.

Reply:
De Toulouse aussi ;) Nos podèm comprendre en catalan coma en occitan :P
Donc il faut que je sorte les variables représentées par des :attributes dans validation-inline pa rapport à validation.php

Reply:
Regarde les fichiers en français et tu comprendras de suite la différence